src/synopsys-action/tools-parameter.ts

```diff
@@ -270,6 +270,28 @@ export class SynopsysToolsParameter {
       blackduckData.data.blackduck.automation.prcomment = false
     }
 
+    if (parseToBoolean(inputs.BLACKDUCK_REPORTS_SARIF_CREATE)) {
+      const sarifReportFilterSeverities: string[] = []
+      if (inputs.BLACKDUCK_REPORTS_SARIF_SEVERITIES != null && inputs.BLACKDUCK_REPORTS_SARIF_SEVERITIES.length > 0) {
```

Comment:
We should check for empty for BLACKDUCK_REPORTS_SARIF_SEVERITIES as in input.ts we are defaulting the value to empty if data is not there.

Reply:
updated